lib/shelley/src/Cardano/Wallet/Shelley/Network.hs

```diff
@@ -381,6 +381,9 @@ withNetworkLayer tr np addrInfo versionData action = do
                 (optimumNumberOfPools pparams)
                 rewardMap
                 stakeMap
+        liftIO $ traceWith tr $ MsgFetchedNodePoolLsqDataSummary
+            (Map.size stakeMap)
+            (Map.size rewardMap)
         liftIO $ traceWith tr $ MsgFetchedNodePoolLsqData res
```

Comment:
That's one way to do it, what about using only a single message, and depending on the severity, show either the full result, or a summary :) ?

Reply:
That was how I first imagined it, but I don't see how.

We need to _provide_ a severity for `MsgFetchedNodePoolLsqData` given the contents of the message. We don't know the global severity level.

Reply:
Hmmm... right, the `ToText` instance is actually not aware of the severity!